torch/onnx/symbolic.py

```diff
@@ -170,6 +170,10 @@ def neg(g, self):
     return g.op("Neg", self)
 
 
+def sqrt(g, self):
+    return g.op('Sqrt', self)
```

Comment:
nit: should not it be double quotes, like all other operations?

Reply:
Done.